ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetWork.java

```diff
@@ -54,7 +54,8 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
       final String[] result = {
         "0x" + BaseEncoding.base16().lowerCase().encode(rawResult.getPrePowHash()),
         "0x" + BaseEncoding.base16().lowerCase().encode(dagSeed),
-        rawResult.getTarget().toHexString()
+        rawResult.getTarget().toHexString(),
+        "0x" + Long.toHexString(rawResult.getBlockNumber())
```

Comment:
Do we have any confirmation that were we to add the block number it should be slotted in the fourth array position?  For example, documentation or other client implementations?

Reply:
Yeah i have confirmation from Geth implementation